flood_api/routers/flood.py

```diff
@@ -22,7 +22,11 @@
 
 
 @router.get(
-    "/summary", response_model=SummaryResponseModel, response_class=JSONResponse
+    "/summary",
+    summary="Get summary forecast",
+    description="Returns a summary forecast of the next 30 days for the given location",
+    response_model=SummaryResponseModel,
```

Comment:
I don't think this is necessary since the function signature is already `SummaryResponseModel`

Reply:
It's done here: https://github.com/openearthplatforminitiative/weather-api/blob/582cacc34ab3b612c103f516b37d4b7a010184a5/weather_api/routes/weather_routes.py#L18

Reply:
I think it's redundant, but maybe @kstigen disagrees

Reply:
I thought it was for Swagger UI documentation..

Reply:
FastAPI is smart enough to use the function signature for Swagger UI

Reply:
I agree with @dymbe. If it is unnecessary, then we remove it. I should remove it from the weather-api as well.